tools/uprev-rdg-storage-format-version-worker/uprev-rdg-storage-format-version-worker.cpp

```diff
@@ -0,0 +1,79 @@
+#include <string>
+
+#include <boost/filesystem.hpp>
+
+#include "katana/Logging.h"
+#include "katana/PropertyGraph.h"
+#include "katana/SharedMemSys.h"
+#include "katana/analytics/Utils.h"
+#include "llvm/Support/CommandLine.h"
+#include "stdio.h"
+#include "tsuba/RDG.h"
+
+namespace cll = llvm::cl;
+namespace fs = boost::filesystem;
+
+/* usage: ./rdg-uprev-storage-format-version.cpp <input-rdg> <output-path>
+ *
+ * loads the given input rdg and stores it in the output-path
+ * validates that the input rdg and the output rdg match
+ * used to uprev the testing inputs
+ */
+
+static cll::opt<std::string> InputFile(
+    cll::Positional, cll::desc("<input rdg file>"), cll::Required);
+
+static cll::opt<std::string> OutputFile(
+    cll::Positional, cll::desc("<output rdg file>"), cll::Required);
+
+katana::PropertyGraph
+LoadGraph(const std::string& rdg_file) {
+  KATANA_LOG_ASSERT(!rdg_file.empty());
+  auto g_res = katana::PropertyGraph::Make(rdg_file, tsuba::RDGLoadOptions());
```

Comment:
We can't do this, can we?  It probably doesn't matter.
`auto g_res = std::move(KATANA_CHECKED(katana::PropertyGraph::Make(rdg_file, tsuba::RDGLoadOptions())));`

Reply:
yeah, we likely can.